docs/docs/gatsby-style-guide.md

```diff
@@ -83,13 +83,13 @@ repository.
 
 # Running the Guide locally on your own computer
 
-Finally, if you want to run a version of the guides repository locally, follow
+Finally, if you want to run a version of the Guides repository locally, follow
 these steps:
 
-1. Ensure you have the yarn package manager installed `npm install -g yarn`
-2. Install the Gatsby cli `yarn add --global gatsby-cli`
-3. Follow the steps above to fork and clone the Gatsby repo.
-4. Enter the `www` directory where gatsbyjs.org is and install its dependencies `yarn install` then run `gatsby develop`.
+1. Install the Gatsby command-line tool `npm install --global gatsby-cli`.
```

Comment:
For our internal stuff, we do want people to use Yarn as we commit yarn.lock files.